### PR TITLE
class display: Remove method "AddContent", because it don't have any functionality

### PR DESCRIPTION
**Pull request by andygrunwald**
The method `AddContent` has no body and is doing nothing here.

### CODE REVIEW

File: modules/noc/device_change.php [hunk 1]
```diff
@@ -70,10 +70,9 @@
             $dsp->AddTextFieldRow("device_read", t('Read-Community'), $device_read, $noc_error['device_read']);
             $dsp->AddTextFieldRow("device_write", t('Write-Community'), $device_write, $noc_error['device_write']);
         
-            $dsp->AddFormSubmitRow(t('�ndern'));
+            $dsp->AddFormSubmitRow(t('�ndern'));
```
Comment (M4LuZ) [inline modules/noc/device_change.php]: Seems to be a character encoding issue here

File: modules/noc/device_details.php [hunk 2]
```diff
@@ -219,13 +219,12 @@
 
             $ports .= "</tr>";
             $smarty->assign('ports', $ports);
-            $changebutton = $dsp->FetchSpanButton(t('Zur�ck'), "index.php?mod=noc&action=show_device");
+            $changebutton = $dsp->FetchSpanButton(t('Zur�ck'), "index.php?mod=noc&action=show_device");
```
Comment (M4LuZ) [inline modules/noc/device_details.php]: aaand here

File: modules/noc/port_details.php [hunk 3]
```diff
@@ -36,7 +36,7 @@
                     break;
             }
         
-        // We assume that of course 1 PetaByte = 1024 Terabyte = 1024� Gigabyte = 1024� Megabyte = 1024 * 1024 * 1024 * 1024 Kilobyte = 1024 * 1024 * 1024 * 1024 * 1024 Byte = 1024 * 1024 * 1024 * 1024 * 1024 * 8 Bit
+        // We assume that of course 1 PetaByte = 1024 Terabyte = 1024� Gigabyte = 1024� Megabyte = 1024 * 1024 * 1024 * 1024 Kilobyte = 1024 * 1024 * 1024 * 1024 * 1024 Byte = 1024 * 1024 * 1024 * 1024 * 1024 * 8 Bit
```
Comment (M4LuZ) [inline modules/noc/port_details.php]: three is the lucky charm ;)

File: modules/stats/search_engines.php [hunk 4]
```diff
@@ -27,9 +27,8 @@
 
 #$ms2->AddIconField('details', 'index.php?mod=news&action=comment&newsid=', t('Details'));
 #if ($auth['type'] >= 2) $ms2->AddIconField('edit', 'index.php?mod=news&action=change&step=2&newsid=', t('Editieren'));
-#if ($auth['type'] >= 3) $ms2->AddIconField('delete', 'index.php?mod=news&action=delete&step=2&newsid=', t('L�schen'));
+#if ($auth['type'] >= 3) $ms2->AddIconField('delete', 'index.php?mod=news&action=delete&step=2&newsid=', t('L�schen'));
```
Comment (M4LuZ) [inline modules/stats/search_engines.php]: may the fourth be with you?

File: modules/equipment/admin.php [hunk 5]
```diff
@@ -4,8 +4,7 @@
...
-    $dsp->AddSingleRow(t('Dieses Modul erm�glicht es einen <a href="http://www.orgapage.net" traget="_blank">Orgapage.Net<a>-EquipmentShop sehr einfach in eine LanSuite-Webseite zu integrieren.
-    <br><br>In der <a href="http://www.orgapage.net/pages/equip/submit.php" traget="_blank">Equipmentliste auf Orgapage.Net</a> kannst du einen solchen Shop erstellen, falls du dies noch nicht getan hast. Von diesem werden dann die Daten ausgelesen, die hier in diesem Modul pr�sentiert werden. Durch das Eintragen wird das von dir angebotene Equipment auch in der Equipmentliste auf Orgapage angezeigt, so dass weitere Besucher evtl. auf deinen Shop aufmerksam werden.
-    <br><br>Einen vorhandenen Shop kannst du unter <a href="http://orgapage.de/pages/equip/shops/admin/index.php" traget="_blank">http://orgapage.de/pages/equip/shops/admin/index.php/</a> verwalten (Bestellungen einsehen / auf reserviert schalten / l�schen / ...).'));
-    $dsp->AddContent();
+    $dsp->AddSingleRow(t('Dieses Modul erm�glicht es einen <a href="http://www.orgapage.net" traget="_blank">Orgapage.Net<a>-EquipmentShop sehr einfach in eine LanSuite-Webseite zu integrieren.
```
Comment (M4LuZ) [inline modules/equipment/admin.php]: Missed this one on first try

**Comment by andygrunwald** [comment]
@M4LuZ Fixed.
While opening these files, my IDE showed me that the files had a wrong encoding.
After saving they were transformed to UTF-8. That caused the errors here.
Fixed those and merged the master back into this branch.

File: modules/install/adminaccount.php [hunk 6]
```diff
@@ -25,7 +25,7 @@
 					type = '3'
 					", $_POST["email"], md5($_POST["password"]));
             $userid = $db->insert_id();
-            // Admin zur Party hinzuf�gen
+            // Admin zur Party hinzuf�gen
```
Comment (M4LuZ) [inline modules/install/adminaccount.php]: Missed one ;)

File: modules/irc/show.php [hunk 7]
```diff
@@ -22,8 +22,7 @@
     $dsp->AddTextFieldRow("firstname", t('Vorname'), $_POST["firstname"], "");
     $dsp->AddTextFieldRow("name", t('Nachname'), $_POST["name"], "");
 
-    $dsp->AddFormSubmitRow(t('Hinzuf�gen'));
-    $dsp->AddContent();
+    $dsp->AddFormSubmitRow(t('Hinzuf�gen'));
```
Comment (M4LuZ) [inline modules/irc/show.php]: Missed two

File: modules/stats/database.php [hunk 8]
```diff
@@ -16,7 +16,7 @@
 
 $hostname = explode(' ', $db->get_host_info());
 
-$dsp->NewContent(t('Datenbank Statistik'), t('Auf dieser Seite erh�lst du Informationen &uuml;ber die Datenbank und aktuelle Leistungsdaten.'));
+$dsp->NewContent(t('Datenbank Statistik'), t('Auf dieser Seite erh�lst du Informationen &uuml;ber die Datenbank und aktuelle Leistungsdaten.'));
```
Comment (M4LuZ) [inline modules/stats/database.php]: three

File: modules/install/adminaccount.php (hunk 6 above)
Comment (andygrunwald) [reply modules/install/adminaccount.php]: Fixed.

File: modules/irc/show.php (hunk 7 above)
Comment (andygrunwald) [reply modules/irc/show.php]: Fixed.

File: modules/stats/database.php (hunk 8 above)
Comment (andygrunwald) [reply modules/stats/database.php]: Fixed.

**Pull request merged**